uber/models.py

```diff
@@ -1338,6 +1345,16 @@ def requested(self, department):
     def assigned_to(self, department):
         return int(department or 0) in self.assigned_depts_ints
 
+    def trusted_in(self, department):
+        return int(department or 0) in self.trusted_depts_ints
+
+    def assigned_and_trusted_in(self, department):
```

Comment:
hey, on reviewing this, I agree with your comment and decided to take it out to make things simpler.  I think on my first pass I figured a bunch of places were going to use ```assigned_and_trusted_in()``` but it ended up being just one, so I ripped it out.